### ISSUE TITLE
Distant shore foam jitters and flickers. 

### ISSUE THREAD

**Issue by andrew-sidereal** [crest-ocean/crest#691]
**Describe the bug**
When viewing distant shorelines, the shoreline foam flickers and jitters. You can see it in this video: https://www.youtube.com/watch?v=EEBr6L6NYkg&feature=youtu.be

Built-in Pipeline, 2019.4.4

**Comment by andrew-sidereal** [comment]
It's extremely evident in the central pond area, but it's actually an issue all around the shore if you look closely.

**Comment by andrew-sidereal** [comment]
It doesn't just seem like LOD or such, it seems to rapidly switch back and forth between different foam positions, creating a flickering effect.

**Comment by daleeidd** [comment]
Is this reproducible in the main example scene?

**Comment by daleeidd** [comment]
Someone else has experiences this problem on Discord. They are using Enviro. Are you also using it? It might be related.

**Comment by andrew-sidereal** [comment]
Yes, I'm using Enviro. Glad you have a lead! Eager for a resolution!

**Comment by huwb** [comment]
Would it be possible to test with enviro disabled, to confirm that is the issue?

**Comment by daleeidd** [comment]
From user Jamison on Discord:

> So I was able to get rid of the flickering by turning off the Anti-Aliasing on the Post Processing component added by Enviro. I'm still working on adding Anti-Aliasing that doesn't cause flicker for me. If I use FXAA or SMAA instead of TAA the problem goes away

**Comment by andrew-sidereal** [comment]
I believe I have the solution -- I am still using TAA for Anti-Aliasing, as FXAA and SMAA aren't feasible in my case. However, by lowering the "Jitter Spread" on the Camera's Post Processing "Anti-Aliasing" section from the default of 0.75 to 0.35, the shoreline jittering effect has been eliminated.

**Comment by daleeidd** [comment]
Thanks. Glad to have a workaround. I believe (hope) this can be properly fixed by supporting motion vectors. It is on the list, but not a high priority.
